caikit/core/__init__.py

```diff
@@ -20,14 +20,16 @@
 # disable warnings, initialize the JVM and configure logging in a specific order
 # pylint: disable=wrong-import-position,wrong-import-order
 
+# NOTE: There are cyclic imports due to the "import *"s here, when modules then
+# "import core"
+
 # Standard
 # We're filtering (most) warnings for now
 import warnings as _warnings
 
 _warnings.filterwarnings("ignore")
 
 # Local
-# must import toolkit first since we need alog to be set up before it is used
```

Comment:
alog at one point was imported through core.toolkit but no longer

Reply:
Yes! This has been a long time coming